### PR TITLE
Print assembly which was not properly resolved

### PR DESCRIPTION
**Pull request by kant2002**
See #3849

### CODE REVIEW

**Comment by dellis1972** [comment]
Thanks for the contribution. I think this looks like a reasonable thing to do in this case.

File: src/Xamarin.Android.Build.Tasks/Tasks/ResolveAssemblies.cs [hunk 1]
```diff
@@ -90,7 +90,7 @@ void Execute (MetadataResolver resolver)
 						if (lockFile != null)
 							resolved_assembly = ResolveRuntimeAssemblyForReferenceAssembly (lockFile, assembly.ItemSpec);
 						if (lockFile == null || resolved_assembly == null) {
-							LogCodedWarning ("XA0107", resolved_assembly, 0, "Ignoring {0} as it is a Reference Assembly", resolved_assembly);
+							LogCodedWarning ("XA0107", resolved_assembly, 0, "Ignoring {0} as it is a Reference Assembly", resolved_assembly ?? assembly.ItemSpec);
```
Comment (jonpryor) [inline src/Xamarin.Android.Build.Tasks/Tasks/ResolveAssemblies.cs]: Close, but not quite; if `resolved_assembly` is `null`, which is what we're assuming will cause the assembly name to be empty, then that means that the `file` parameter provide to `LogCodedWarning()` will *also* be null/empty, which presumably is not quite useful within the IDE.

**Pull request merged**